### PR TITLE
APPCAFE-05: Firebase

### PR DESCRIPTION
**Pull request by 1699kratos**
## ISSUE: 
* https://trello.com/c/sPQdnYrW

## FIX:
 * Followed the documentation that was in the card 

## EVIDENCES:
image
image

### CODE REVIEW

**Review by hachiosaki — CHANGES_REQUESTED** [review]
please, could change the evidence? I prefer a screenshot of the successful connection or a class with the import... also you could do a test where you upload some strings to the database

File: CafePrices/Podfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for CafePrices
+  
+pod 'Firebase/Database'
+pod 'Firebase/Auth'
+pod 'Firebase/Storage'
+pod 'FirebaseUI/Storage'
+pod 'Nuke', '~> 9.0'
```
Comment (hachiosaki) [inline CafePrices/Podfile:14]: why did you import nuke?

**Review by hachiosaki — CHANGES_REQUESTED** [review]

File: CafePrices/Podfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+target 'CafePrices' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for CafePrices
+  
+pod 'Firebase/Database'
+pod 'Firebase/Auth'
+pod 'Firebase/Storage'
```
Comment (hachiosaki) [inline CafePrices/Podfile:12]: what is the difference between Firebase/Storage and FirebaseUI/Storage, are both of them necessary?

**Review by hachiosaki — CHANGES_REQUESTED** [review]

File: CafePrices/Podfile (hunk 1 above)
Comment (1699kratos) [reply CafePrices/Podfile:14]: I thought i will use it sorry

File: CafePrices/Podfile (hunk 2 above)
Comment (1699kratos) [reply CafePrices/Podfile:12]: Firebase is a platform that you use to build web and mobile applications. It consists of a suite of cloud services and a set of SDKs (and in some cases REST APIs) to access those services.

FirebaseUI is a set of libraries that build on top of the Firebase SDKs to make binding to user interface elements easier.

Specifically: FirebaseUI for Android and for iOS wrap the Database and Authentication SDKs of Firebase to make it easy to use the services in Android and iOS applications.

FirebaseUI for web is a newer entrant to the field and only wraps the Authentication SDK. To make it easy using the database in your web app, look for a dedicated library for your preferred web platform: AngularFire, AngularFire2, ReactFire, EmberFire. The names probably speak for themselves in what web library they target.

Comment (hachiosaki) [reply CafePrices/Podfile:12]: ok lets let it there

**Review by hachiosaki — APPROVED** [review]
LGTM!

**Pull request merged**